data/www/app/config/DependencyInjection/repositories.yml

```diff
@@ -0,0 +1,54 @@
+services:
+    database:
+        class: 'App\Repository\Database'
+        parameters:
+            $dsn: 'mysql:host=localhost;dbname=test;charset=utf8'
```

Comment:
We could use a config file actually

Reply:
Pour une ligne de conf c'est relou de faire un fichier + le delarer dans le kernel pour qu'il soit load et tout